docs/source/_templates/page.html

```diff
@@ -94,16 +101,22 @@
           <span>{% trans %}Back to top{% endtrans %}</span>
         </a>
         <div class="content-icon-container">
-          {% if theme_top_of_page_button == "edit" -%}
-          {%- include "components/edit-this-page.html" with context -%}
-          {%- elif theme_top_of_page_button != None -%}
-          {{ warning("Got an unsupported value for 'top_of_page_button'") }}
-          {%- endif -%}
+          {%- set theme_top_of_page_buttons = [] -%}
+          {% for button in theme_top_of_page_buttons -%}
+            {% if button == "view" %}
+            {%- include "components/view-this-page.html" with context -%}
+            {% elif button == "edit" %}
+            {%- include "components/edit-this-page.html" with context -%}
+            {% else %}
+            {{ warning("Got an unsupported value in 'top_of_page_buttons' for theme configuration") }}
+            {% endif %}
+          {%- endfor -%}
           {#- Theme toggle -#}
           <div class="theme-toggle-container theme-toggle-content">
             <button class="theme-toggle">
               <div class="visually-hidden">Toggle Light / Dark / Auto color theme</div>
-              <svg class="theme-icon-when-auto"><use href="#svg-sun-half"></use></svg>
+              <svg class="theme-icon-when-auto-light"><use href="#svg-sun-half"></use></svg>
+              <svg class="theme-icon-when-auto-dark"><use href="#svg-sun-half"></use></svg>
```

Comment:
Aligning with furo ([ref](https://github.com/pradyunsg/furo/blob/1bbf4ee4f65c488b9a0f577b8205255338a64c4d/src/furo/theme/furo/page.html#L108-L109)):
```suggestion
              <svg class="theme-icon-when-auto-light"><use href="#svg-sun-with-moon"></use></svg>
              <svg class="theme-icon-when-auto-dark"><use href="#svg-moon-with-sun"></use></svg>
```

Reply:
Added!